### PR TITLE
add template for job submitting

### PR DESCRIPTION
**Pull request by sophiesongge**

### CODE REVIEW

**Comment by sophiesongge** [comment]
retest this please

**Comment by sophiesongge** [comment]
retest this please

**Comment by sophiesongge** [comment]
retest this please

**Comment by sophiesongge** [comment]
retest this please

File: app/templates/SubmitJobAndWait.xml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+     maxNumberOfExecution="2"
>
+  <taskFlow>
+    <task name="SubmitJobAndWait">
+      <description>
+        <![CDATA[ The simplest task, ran by a groovy engine. ]]>
+      </description>
+      <variables>
+        <variable name="workflowURL" value="http://trydev.activeeon.com:8080/workflow-catalog/buckets/1/workflows/4?alt=xml" inherited="false" model="String"/>
+      </variables>
```
Comment (marcocast) [inline app/templates/SubmitJobAndWait.xml]: can we use the https address and is it going to work for try?and locally?

File: app/templates/SubmitJobNoWait.xml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+     maxNumberOfExecution="2"
>
+  <taskFlow>
+    <task name="SubmitJobNoWait">
+      <description>
+        <![CDATA[ The simplest task, ran by a groovy engine. ]]>
+      </description>
+      <variables>
+        <variable name="workflowURL" value="http://trydev.activeeon.com:8080/workflow-catalog/buckets/1/workflows/4?alt=xml" inherited="false" model="String"/>
+      </variables>
```
Comment (marcocast) [inline app/templates/SubmitJobNoWait.xml]: same as the other one

**Comment by marcocast** [comment]
I cannot see the icons images, but maybe it s a github problem. Anyway i added a couple of comments

File: app/templates/SubmitJobNoWait.xml (hunk 2 above)
Comment (marcocast) [inline app/templates/SubmitJobNoWait.xml]: same as the other one

**Comment by sophiesongge** [comment]
This one is under control menu, in order to have consistency with other templates, I put the logos to empty too.

File: app/templates/SubmitJobAndWait.xml (hunk 1 above)
Comment (bamedro) [inline app/templates/SubmitJobAndWait.xml]: trydev is an internal platform for testing purpose. URL to this platform should not appear in a release.
In this case, I would put:
http://try.activeeon.com:8080/workflow-catalog/buckets/1/workflows/4?alt=xml
something like:
http://repo.domain.com/workflow.xml
is another option.
But let's go for the first one.

**Pull request merged**